src/shared/api/server/summary/all.ts

```diff
@@ -59,17 +71,8 @@ export const GET = async (req: NextRequest): Promise<NextResponse<SummariesRespo
     const offset = Number(searchParams.get('offset')) || 0;
     const window = searchParams.get('durationWindow');
 
-    if (!window || !isDurationWindow(window)) {
-      return NextResponse.json(
-        {
-          error: `durationWindow missing or invalid window. Options: ${durationWindows.join(', ')}`,
-        },
-        { status: 400 },
-      );
-    }
-
     const result = await getAllSummaries({
-      window,
+      window: window && isDurationWindow(window) ? window : '1d',
```

Comment:
suggestion: if someone passes a window, I'd do validation+throw if `isDurationWindow()` returns a false instead of defaulting the query to `1d`. Or... we can just require the duration window passed by the client too.

Reply:
agree, updated